### DISCUSSION TITLE
UI for adding tracks to a collection

### DISCUSSION THREAD

**Discussion by jhurlbut** [okio-ai/nendo-platform#22] [Ideas]
I don't see any feature in the UI to add tracks to a collection after import or analysis

**Comment by faradox** [comment]
Upon import:

![2024-02-25_13-07](https://github.com/okio-ai/nendo-platform/assets/5659844/43f89661-9e88-4d47-97fd-95099805dacc)

Upon analysis (only for tools that create new tracks):

![2024-02-25_13-07_1](https://github.com/okio-ai/nendo-platform/assets/5659844/51c0417f-5d8a-41a4-bfad-43a0b21f30c8)

Directly from library view:

![2024-02-25_13-32](https://github.com/okio-ai/nendo-platform/assets/5659844/e5cf3261-c159-42d3-a5f3-29361d015ae6)

I hope this answers your question? :)
